app/utils/requests/project/queries.ts

```diff
@@ -43,6 +43,33 @@ export const ProjectFragment = graphql(
   [InnoUserFragment],
 );
 
+// search in title, summary, author, description.text, description.tags.tag
+export const SearchProjectsQuery = graphql(
```

Comment:
get query and request from [PR#107](https://github.com/openkfw/innoverse/pull/107/files)